src/libslic3r/GCode/GCodeProcessor.hpp

```diff
@@ -261,7 +261,7 @@ namespace Slic3r {
             Last_Line_M73_Placeholder,
             Estimated_Printing_Time_Placeholder,
             Total_Layer_Number_Placeholder,
-            Tool_Change,
+            MANUAL_TOOL_CHANGE,
```

Comment:
Wouldn't it be better to keep the tag variable name in camel case to match the format of the other tags? For example, Manual_Tool_Change. (To be clear, I am perfectly fine with how the tag presents in gcode)